README.md

```diff
@@ -28,7 +28,7 @@ Fake tokens are used just like JWTs: As bearer tokens in the Authorization heade
 
 Here is an example fake-token that can be used as the bearer authorization token.
 ```
-eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19
+eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19
```

Comment:
Keep the decoded version in mind as well

Reply:
`{"sub":"18159713-5d4e-4ad5-94ad-fbb6bb147984","email":"testine.test@helpwave.de","name":"Testine Test","nickname":"testine.test","organizations":["3b25c6f5-4705-4074-9fc6-a50c28eba406"]}`